backend/src/controller/handleError.ts

```diff
@@ -22,7 +22,8 @@ function handleChatError(
 		reply: errorMsg,
 		isError: true,
 	};
-	res.status(statusCode).send(updatedChatResponse);
+	res.status(statusCode);
+	res.send(updatedChatResponse);
```

Comment:
What's the difference here?

Reply:
oh yeah, I did that while experimenting with mocking, and forgot to change it back

Reply:
reverted